.yarnrc

```diff
@@ -0,0 +1,2 @@
+
+save-prefix false
```

Comment:
If we're using semantic-release, we should be able to save-prefix `^` which is the default. Or does this do something different with yarn that npm?